app/controllers/sessions_controller.rb

```diff
@@ -14,6 +14,11 @@ def create
     redirect_params = {}
     if user.nil?
       redirect_params = { alert: 'Login or password incorrect. Try again.' }
+    elsif user.email_confirmed_at.nil?
+      non_confirmed user
+      @user = user
+      redirect_to email_confirmation_request_path
+      return
```

Comment:
Should we have the return on the same line as the redirect
`return redirect_to email_confirmation_request_path`